rust+wasm/{{project-name}}-wasm/README.md

```diff
@@ -94,7 +94,7 @@ Javascript-wrapper file, the {{project-name}}-wasm README (and version), and a
 For running tests in the current directly, use this command:
 
 ```console
-wasm-pack test
+wasm-pack test --node | --headless --*
```

Comment:
Having trouble running this command. Should there be a `|` in it?

When I remove the pipe, it produces this error:

> Error: The `--headless` flag only applies to browser tests. Node does not provide a UI, so it doesn't make sense to talk about a headless version of Node tests.

Reply:
it's not a real command haha.

Reply:
i'll fix up the readme to make the explicit ones haha.